<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordDisplay</class>
 <widget class="QMainWindow" name="PasswordDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1113</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1113</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="PasswordDisplayLE">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>225</y>
      <width>425</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="proceedPB">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>500</y>
      <width>200</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(170, 170, 127)</string>
    </property>
    <property name="text">
     <string>Proceed</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1111</width>
      <height>851</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:url(:/back/BACKGROUND.png) 0 0 0 0 stertch stertch</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>PasswordDisplayLE</zorder>
   <zorder>proceedPB</zorder>
  </widget>
 </widget>
 <resources>
  <include location="r3.qrc"/>
 </resources>
 <connections/>
</ui>
